### PR TITLE
Added some tests in test_learning.py

### PR DESCRIPTION
**Pull request by theSparta**
Added tests for the `error` functions in `learning.py` as required by #286 .
@reachtarunhere, What kind of other tests can be added?

### CODE REVIEW

File: tests/test_learning.py [hunk 1]
```diff
@@ -12,3 +12,31 @@ def test_weighted_mode():
 
 def test_weighted_replicate():
     assert weighted_replicate('ABC', [1, 2, 1], 4) == ['A', 'B', 'B', 'C']
+
```
Comment (sampadsaha5) [inline tests/test_learning.py]: I really appreciate your work. This is just a small mistake.
You are required to import `rms_error()`, `manhattan_distance()`, `mean_boolean_error()` and `mean_error()` from learning.

Comment (theSparta) [inline tests/test_learning.py]: Done!

**Comment by norvig** [comment]
Now there is a conflict in learning.py

**Comment by theSparta** [comment]
Actually I have already resolved the issue #295 in this pull request in addition to adding the tests. I referenced that commit in the issue too.

**Comment by vidursatija** [comment]
I didn't know you resolved the issue #295 in your pull request. I independently resolved it. Sorry for any inconvenience caused.

**Comment by Chipe1** [comment]
@theSparta Could you break this PR into 2 parts, 1 which resolves the mistake in count() and one which adds test cases after resolving the conflicts. You can aslo uncomment the mean_boolean_error() test cases as the bug seems to be fixed

**Pull request merged**